### PR TITLE
Add alma webhook monitor, ruby edition

### PR DESCRIPTION
**Pull request by hackartisan**
* port code from js into ruby:
  * https://github.com/pulibrary/alma_experimentation/pull/8
  * https://github.com/pulibrary/alma_experimentation/pull/33
* Add a bunch of documentation in a sub-README
* Use an alma-event namespace in AWS
* Use a production stage

advances #859

Co-authored-by: Eliot Jordan <eliotj@princeton.edu>

### CODE REVIEW

File: scratch [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (tpendragon) [inline scratch:1]: What is this file? Accidentally added?

File: webhook_monitor/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+## Alma Webhook Setup
```
Comment (tpendragon) [inline webhook_monitor/README.md:1]: Could we put these sections at the bottom with a note saying that this was an initial setup that you don't have to do for every dev or something?

**Review by tpendragon — CHANGES_REQUESTED** [review]
Just some small things. Looks great! This is already wired up to Alma?

File: scratch (hunk 1 above)
Comment (hackartisan) [reply scratch]: Yes accident thanks! (sample of the json you get out of the alma full dump event)

File: webhook_monitor/README.md (hunk 2 above)
Comment (hackartisan) [reply webhook_monitor/README.md]: Yes good idea

**Comment by hackartisan** [comment]
@tpendragon ready for re-review

**Review by tpendragon — APPROVED** [review]

**Pull request merged**